tsconfig.json

```diff
@@ -22,8 +22,7 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true
+    "isolatedModules": true
   },
   "include": ["src"],
   "exclude": ["node_modules", "dist", "example"]
```

Comment:
can we get rid of `example` already? :)

Reply:
yes, example was originally a directory with library examples

Reply:
Done